TRENZ.Docs.API/Services/MarkdownFileProcessingService.cs

```diff
@@ -22,7 +21,7 @@ public MarkdownFileProcessingService(ILogger<MarkdownFileProcessingService> logg
     }
     
     /// <inheritdoc />
-    public async IAsyncEnumerable<IndexFile> ProcessAsync(IAsyncEnumerable<ISourceFile> files, [EnumeratorCancellation] CancellationToken cancellationToken = default)
+    public async IAsyncEnumerable<IndexFile> ProcessAsync(IEnumerable<ISourceFile> files, [EnumeratorCancellation] CancellationToken cancellationToken = default)
```

Comment:
isn't `EnumeratorCancellation` moot now?

Reply:
No, the return type is still an `IAsyncEnumerable`